### PR TITLE
feat(http): Add `getHttpSafe()` method

### PR DESCRIPTION
**Pull request by zharinov**
## Changes

- `getJsonSafe` returns `ResultPromise<T, HttpError | ZodError | EmptyResultError>`
- `EmptyResultError` is returned in case of response (possibly, transformed) containing `null` or an empty string.

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

**Review by viceice — COMMENTED** [review]
blocked by other PR. 🤗

**Review by viceice — COMMENTED** [review]
otherwise LGTM

File: lib/util/http/index.ts [hunk 1]
```diff
@@ -348,6 +352,38 @@ export class Http<Opts extends HttpOptions = HttpOptions> {
     return this.requestJson<ResT>('get', args);
   }
 
+  getJsonSafe<ResT>(
```
[the remark this answers is not in the document]
Comment (zharinov) [reply lib/util/http/index.ts:355]: Yes, I think it's a good idea

**Review by viceice — COMMENTED** [review]
lint error

**Review by viceice — APPROVED** [review]

**Pull request merged**